uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-files-tab.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'common/moment',
+  'leave-absences/shared/modules/components',
+  'common/services/hr-settings'
+], function (_, moment, components) {
+  components.component('leaveRequestPopupFilesTab', {
+    bindings: {
+      // canManage: '<',
+      mode: '<',
+      request: '<'
+    },
+    templateUrl: ['shared-settings', function (sharedSettings) {
+      return sharedSettings.sharedPathTpl + 'directives/leave-request-popup/leave-request-popup-files-tab.html';
+    }],
+    controllerAs: 'vm',
+    controller: ['$log', 'HR_settings', 'shared-settings', 'Contact', controller]
+  });
+
+  function controller ($log, HRSettings, sharedSettings, Contact) {
+    $log.debug('Component: leave-request-popup-files-tab');
+
+    var filesContacts = [];
+    var vm = Object.create(this);
+    vm.supportedFileTypes = '';
+    vm.today = Date.now();
+    vm.userDateFormatWithTime = HRSettings.DATE_FORMAT + ' HH:mm';
+    vm.userDateFormat = HRSettings.DATE_FORMAT;
+
+    (function init () {
```

Comment:
the `init` function should be between the public (`vm.xyz`) and private (`function xyz`) methods, like for any other component